pyproject.toml

```diff
@@ -27,15 +27,15 @@ license = "BSD-2-Clause"
 [tool.poetry.dependencies]
 python = ">=3.6.1,<3.9"
 django = ">=2.2.12,<4.0"
-notifications-python-client = "5.4.1"
+notifications-python-client = "^6.1.0"
 
 [tool.poetry.dev-dependencies]
 black = "^19.10b0"
 isort = "^4.3.21"
 flake8 = "^3.8.2"
 pre-commit = "^2.4.0"
 detect-secrets = "0.13.0"
-factory-boy = "^3.0"
+factory-boy = "^3.2.0"
```

Comment:
**Comment**: The additional changes to Factory Boy were due to spurious errors I was getting from Poetry that it could not resolve the version "^3.0". I uninstalled and reinstalled it, and this is the version string it came up with.